moneta-convert/moneta-convert-base/src/test/java/org/javamoney/moneta/convert/ConversionOperatorsTest.java

```diff
@@ -43,7 +43,7 @@ public void shouldExchangeCurrencyPositiveValue() {
 		MonetaryAmount result = ConversionOperators.exchange(real).apply(money);
 		assertNotNull(result);
 		assertEquals(result.getCurrency(), real);
-		assertEquals(Double.valueOf(2.35), result.getNumber().doubleValue());
+		assertEquals(2.35d, result.getNumber().doubleValue());
```

Comment:
maybe it's better to make the `d` literal in uppercase for consistency with `L` for longs

Reply:
Would you do this across the entire code base? Also for floats?

Reply:
I can't say that this is correct style and I guess this should be decided by the project leaders.
But actually I would like to rise another point: most asserEquals() by the Double are checking an amount so maybe we can simplify to make an equals my MonetaryAmount ie.
assertEquals(result, Money.of(2.35, "USD");